WDL/runtime/task.py

```diff
@@ -891,6 +904,7 @@ def run_local_task(
                     outputs, os.path.join(run_dir, "outputs.json"), namespace=task.name
                 )
                 logger.notice("done")  # pyre-fixme
+                cache.put(input_digest, run_dir, outputs)
```

Comment:
We could add some log messages to help us monitor what's happening; perhaps INFO level for when we put something into the cache or make an unsuccessful cache lookup, and a NOTICE-level message when we actually use cached outputs

Reply:
👍